internal/sftp/goclient.go

```diff
@@ -57,6 +57,32 @@ func (c *GoClient) Upload(ctx context.Context, src io.Reader, dest string) (int6
 	return bytes, nil
 }
 
+// Delete removes the data from dest and returns the
```

Comment:
See above suggestion for updating this comment. Also the comment text should be re-wrapped at 80 cols.